diff --git a/flake.nix b/flake.nix
index f0b7054..33552f7 100644
--- a/flake.nix
+++ b/flake.nix
@@ -3,9 +3,6 @@
 
   inputs = {
     nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
-    mynixpkgs = {
-      url = "github:< this is your own nixpkgs >/nixpkgs/master";
-    };
     home-manager = {
       url = "github:nix-community/home-manager/master";
       inputs.nixpkgs.follows = "nixpkgs";
@@ -29,23 +26,11 @@
       ];
     };
 
-    mkNixOSConfigurations = host: let
-      mypkgs = import inputs.mynixpkgs {
-        system = host.arch;
-        config = {
-          allowUnfree = true;
-        };
-      };
-    in nixpkgs.lib.nixosSystem {
+    mkNixOSConfigurations = host: nixpkgs.lib.nixosSystem {
       system = host.arch;
       modules = [
         ./hosts/${host.dir}/configuration.nix
         ./overlays
-        {nixpkgs.overlays = [
-          (final: prev: {
-            sample-package = mypkgs.your-sample-package;
-          })
-        ];}
         home-manager.nixosModules.home-manager
         {
           home-manager.useGlobalPkgs = true;
